applications/dashboard/Gruntfile.js

```diff
@@ -124,7 +124,7 @@ module.exports = function (grunt) {
             src: [
               'bootstrap/LICENSE'
               , 'bootstrap/scss/*.scss'
-              , 'bootstrap/scss/mixins/*.scss'
+              , 'bootstrap/scss/styleUtilities/*.scss'
```

Comment:
Did you intend to touch this? My gut tells me it's find/replace that reached too far.